contracts/dapis/DapiServer.sol

```diff
@@ -842,69 +844,22 @@ contract DapiServer is
         return dapiNameHashToDataFeedId[keccak256(abi.encodePacked(dapiName))];
     }
 
-    /// @notice Reads the data feed with ID
-    /// @param dataFeedId Data feed ID
+    /// @notice Reads the data feed with dAPI name hash
+    /// @param dapiNameHash dAPI name hash
     /// @return value Data feed value
     /// @return timestamp Data feed timestamp
-    function readDataFeedWithId(bytes32 dataFeedId)
+    function readDataFeedWithDapiNameHash(bytes32 dapiNameHash)
         external
         view
         override
         returns (uint224 value, uint32 timestamp)
     {
-        DataFeed storage dataFeed = dataFeeds[dataFeedId];
-        return (dataFeed.value, dataFeed.timestamp);
-    }
-
-    /// @notice Reads the data feed value with ID
-    /// @param dataFeedId Data feed ID
-    /// @return value Data feed value
-    function readDataFeedValueWithId(bytes32 dataFeedId)
```

Comment:
Is the assumption that if they want to read directly from the datafeed they have to deploy the OEV proxy ?

Reply:
No, `dataFeeds` is `public` now so people can still read from that directly. The reason why there was a `readDataFeedValueWithId()` before is that we needed to check for whitelisting, with that gone `readDataFeedValueWithId()` is obsolete.